cmd/jag/commands/util.go

```diff
@@ -375,3 +375,51 @@ func (s *shortEncoder) Encode(v interface{}) error {
 	}
 	return nil
 }
+
+func getWifiCredentials(cmd *cobra.Command) (string, string, error) {
```

Comment:
I wonder if we should provide a way to update/store credentials in the config as part of this?

Do you want to store the credentials [y/N]?
Do you want to update the stored credentials [y/N]?

Reply:
Thought of this too.
Maybe in a different PR.